src/screens/PinFlow/CurrentPinScreen.tsx

```diff
@@ -54,14 +54,13 @@ class CurrentPinScreen extends PureComponent<Props, State> {
     let blockedTimeInMinutes = 1;
 
     if (attempt === 0) {
-      currentDate = currentDate.add(1, 'minute');
-      blockedTimeInMinutes = 1;
+      currentDate = currentDate.add(blockedTimeInMinutes, 'minute');
```

Comment:
just `currentDate.add(blockedTimeInMinutes, 'minute');` instead `currentDate = currentDate.add(blockedTimeInMinutes, 'minute');`

Reply:
I've tried this before, but it does not working, as it's not saving added time, it's returning new object with added time 

> Returns a cloned Day.js object with a specified amount of time added.

https://day.js.org/docs/en/manipulate/add#docsNav

@KedziaPawel

Reply:
ok, my bad than I thought it mutates an object

Reply:
anyway Im gonna refactor it in that way:

```   
 if (attempt === 1) {
      blockedTimeInMinutes = 2;
    } else if (attempt > 1) {
      blockedTimeInMinutes = 10;
    }
    currentDate = currentDate.add(blockedTimeInMinutes, 'minute');
```